test/e2e/single_zone_e2e_test.go

```diff
@@ -0,0 +1,188 @@
+/*
+Copyright 2018 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+    http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package e2e
+
+import (
+	"flag"
+	"fmt"
+	"path/filepath"
+	"strings"
+	"testing"
+
+	"k8s.io/apimachinery/pkg/util/uuid"
+	remote "sigs.k8s.io/gcp-compute-persistent-disk-csi-driver/test/binremote/binremote"
+
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+)
+
+const (
+	network        = "unix"
+	testNamePrefix = "gcepd-csi-e2e-"
+
+	defaultSizeGb    int64 = 5
+	readyState             = "READY"
+	standardDiskType       = "pd-standard"
+	ssdDiskType            = "pd-ssd"
+)
+
+var (
+	client   *csiClient
+	instance *remote.InstanceInfo
+	//gceCloud *gce.CloudProvider
+	nodeID string
+
+	project        = flag.String("project", "", "Project to run tests in")
+	serviceAccount = flag.String("service-account", "", "Service account to bring up instance with")
+	runInProw      = flag.Bool("run-in-prow", false, "If true, use a Boskos loaned project and special CI service accounts and ssh keys")
+)
+
+func TestE2E(t *testing.T) {
```

Comment:
Is the intention that you will be able to run multiple test files in one run?  If so, you may want to move this "once per process" setup into its own file

Reply:
done